server/src-lib/Hasura/Server/Migrate/Version.hs

```diff
@@ -9,10 +9,14 @@ module Hasura.Server.Migrate.Version
 
 import           Hasura.Prelude
 
-import qualified Data.Text      as T
+import qualified Data.Text                  as T
+import           Language.Haskell.TH.Syntax as TH
 
 latestCatalogVersion :: Integer
-latestCatalogVersion = 32
+latestCatalogVersion = 
```

Comment:
Could you add a documentation comment to this binding that explains why this is kept in a separate file? Also, could you update the instructions in `Hasura.Server.Migrate` to tell people to update the `catalog_version` file instead of this module?

Reply:
Done